bundles/EcommerceFrameworkBundle/Tracking/Tracker/GoogleTagManager.php

```diff
@@ -60,7 +60,7 @@ protected function configureOptions(OptionsResolver $resolver)
         parent::configureOptions($resolver);
 
         $resolver->setDefaults([
-            'template_prefix' => 'PimcoreEcommerceFrameworkBundle:Tracking/analytics/tagManager',
+            'template_prefix' => '@PimcoreEcommerceFramework/Tracking/analytics/tagManager',
```

Comment:
Change `'%s:%s.js.twig'` to `'%s/%s.js.twig'` getTemplatePath():
https://github.com/pimcore/pimcore/blob/288477cf6450b9317a75389422ae2975c5e81ef8/bundles/EcommerceFrameworkBundle/Tracking/Tracker.php#L89